gapic/src/main/com/google/gapid/util/Logging.java

```diff
@@ -77,6 +85,10 @@ static LogLevel fromLevel(Level level) {
     setLevel(Level.ALL);
   }};
 
+  private static final Listener NULL_LISTENER = (m) -> {};
```

Comment:
I have a warning on empty blocks, so please add a comment like ```/* do nothing */``` to explicitly state that it is indeed the intent.